File: app/src/main/java/com/karthik/splash/Models/LikePhoto/Photo.java
package com.karthik.splash.Models.LikePhoto;


import com.google.gson.annotations.SerializedName;

import com.karthik.splash.Models.PhotosLists.Links;


public class Photo{

	@SerializedName("urls")
	public Urls urls;

	@SerializedName("color")
	public String color;

	@SerializedName("width")
	public int width;

	@SerializedName("description")
	public String description;

	@SerializedName("links")
	public Links links;

	@SerializedName("id")
	public String id;

	@SerializedName("liked_by_user")
	public boolean likedByUser;

	@SerializedName("height")
	public int height;

	@SerializedName("likes")
	public int likes;
}

File: app/src/main/java/com/karthik/splash/models/likephoto/Photo.kt
package com.karthik.splash.models.likephoto


import com.google.gson.annotations.SerializedName

import com.karthik.splash.models.photoslists.Links


data class Photo(
        @SerializedName("urls") var urls: Urls? = null,
        @SerializedName("color") var color: String? = null,
        @SerializedName("width") var width: Int = 0,
        @SerializedName("description") var description: String? = null,
        @SerializedName("links") var links: Links? = null,
        @SerializedName("id") var id: String? = null,
        @SerializedName("liked_by_user") var likedByUser: Boolean = false,
        @SerializedName("height") var height: Int = 0,
        @SerializedName("likes") var likes: Int = 0
)